### DISCUSSION TITLE
Headings in portfolio page

### DISCUSSION THREAD

**Discussion by ShweataNHegde** [sylhare/Type-on-Strap#354] [Q&A] [answered]
Hi, 

Thanks so much for creating such a nice theme! 

I have very little experience with HTML and liquid. Would be possible to display headings along with the images in the portfolio page?

**Comment by lg-nineoneninethreenine** [comment]
Can you please explain by what you mean?

**Comment by sylhare** [comment]
You mean... 
1. on the portfolio's article (already there when set)
<img width="1113" alt="image" src="https://user-images.githubusercontent.com/20642750/146045720-bb012720-a8fd-42c3-9995-aa4708e465c8.png">

2. on the portfolio's tile (not there for the moment, only as link):
<img width="1572" alt="image" src="https://user-images.githubusercontent.com/20642750/146045903-73e5a1ea-3884-4715-a73b-0203f4eb2661.png">
<img width="296" alt="image" src="https://user-images.githubusercontent.com/20642750/146046336-f4974bac-20b4-4b00-84e4-ec395ab6394f.png">

**Comment by ShweataNHegde** [comment]
Thanks so much for responding @im-coder-lg and @sylhare. 

@sylhare, I meant (2.) on the portfolio tile.

**Comment by sylhare** [comment, marked as answer]
So it's not possible now, maybe I'll think into making it a feature.
To customize it though it's pretty simple, you just need to update [_includes/portfolio.html](https://github.com/sylhare/Type-on-Strap/blob/master/_includes/portfolio.html)

With something like:

```html
<div class="portfolio-grid">
    {% for item in site.portfolio %}
    <div class="portfolio-cell">
        <a class="portfolio-link" data-keyboard="true" href="{{ item.url | relative_url }}">
            <div class="caption" title="{{ item.title }}">
                <div class="caption-content">
                    <i>{{ item.title }}</i>
                </div>
            </div>
            <img alt="" class="" src="{{ item.img | relative_url }}">
        </a>
    </div>
    {% endfor %}
</div>
```

The `{{ item.title }}` is the title of the page that will appear when you hover.
If you want to update the CSS for different size and colors, you can update the `.caption-content` class.

**Comment by ShweataNHegde** [comment]
Got it. Thanks so much for your response!
